cmd/revpn.go

```diff
@@ -58,16 +104,29 @@ func start() (*exec.Cmd, error) {
 		return nil, errors.Wrap(err, "netExtender command failed")
 	}
 
-	merged_output := io.MultiReader(stderr, stdout)
+	merged_output := io.MultiReader(stdout, stderr)
 	scanner := bufio.NewScanner(merged_output)
+	scanner.Split(ScanPrompt)
+	writer := bufio.NewWriter(stdin)
 	for scanner.Scan() {
-		fmt.Println("[netExtender] ", scanner.Text())
-		if strings.HasPrefix(scanner.Text(), "Another NetExtender instance is already running") {
+		// Not sure why these writes are needed, but without we get a lock-up
+		writer.WriteString("\n")
+		writer.Flush()
+		text := scanner.Text()
+		fmt.Println("[netExtender] ", text)
+		if strings.HasPrefix(scanner.Text(), "Do you want to proceed? (Y:Yes, N:No, V:View Certificate)") {
+			writer.WriteString("Y")
+			writer.Flush()
+		}
+		if strings.HasPrefix(text, "Another NetExtender instance is already running") {
 			return nil, errors.New("NetExtender already running; please kill it and try again.")
 		}
-		if strings.HasPrefix(scanner.Text(), "NetExtender connected successfully") {
+		if strings.HasPrefix(text, "NetExtender connected successfully") {
 			return command, nil
 		}
+		if strings.HasSuffix(text, ": ") {
```

Comment:
can you explain this? also, seems a bit open ended...

Reply:
`scanner.Scan()` only returns sth once it finds a full "token". By default, it uses `bufio.SplitLines` to tokenize on `\n`. In our case, we want to be able to detect a partial line (the interactive prompt), so I tokenize on wither `\n` or `: `.

Reply:
I wonder if this will break things down the line if netExtender prints something with `:$` without prompting the user. At the very least I would do two things:
1. add a comment
2. this is idiomatic, move the case that doesn't return an error to last case in the scope.

Reply:
well yes it would, as in "Y:Yes" before I chaged `:` to `: `. It's definitely not fool-proof, but I am guessing the alternative would be to read characters/words and re-assemble them into lines myself...

Reply:
Sorry which case do you mean? I have literally copy-pasted this from `bufio` source

Reply:
see the new comment, that I can include a suggestion